src/react_native/async_storage.cljs

```diff
@@ -0,0 +1,14 @@
+(ns react-native.async-storage
+  (:require ["@react-native-async-storage/async-storage" :default AsyncStorage]))
+
+
+(defn set-item
+  [key value]
+  (-> (.setItem AsyncStorage (clj->js key) (str value))
+      (.catch #(js/console.error (str "Error storing data in async-storage: " %)))))
```

Comment:
Do you think it's better to use `js/alert` ?

Reply:
`js/alert` who show in release mode.